static/app/gettingStartedDocs/flutter/flutter.tsx

```diff
@@ -50,9 +50,15 @@ type Params = DocsParams<PlatformOptions>;
 const isAutoInstall = (params: Params) =>
   params.platformOptions?.installationMode === InstallationMode.AUTO;
 
-const getInstallSnippet = (params: Params) => `
-dependencies:
-  sentry_flutter: ^${getPackageVersion(params, 'sentry.dart.flutter', '7.8.0')}`;
+const getInstallSnippet = ({isSelfHosted, organization, projectSlug}: Params) => {
+  const urlParam = isSelfHosted ? '' : '--saas';
+  return `brew install getsentry/tools/sentry-wizard && sentry-wizard -i flutter ${urlParam} --org ${organization.slug} --project ${projectSlug}`;
+};
+
+const getManualInstallSnippet = (params: Params) => {
+  const version = getPackageVersion(params, 'sentry.dart.flutter', '8.13.2');
+  return `sentry_flutter: ^${version}`;
```

Comment:
are you omitting `dependencies:` intentionally here?

Reply:
what would you expect here? min version of SDK dependencies or what?